blog/_posts/2017-03-01-zinc-blog-post.md

```diff
@@ -0,0 +1,143 @@
+handling](https://github.com/sbt/zinc/issues/150).
+  
+## Zinc 1.0 in action: 7x speedups, an example
+  
+Thanks to class-based dependency and name-hashing heuristics, Zinc is smarter
+when it comes to detecting dependencies between source files. For instance,
+complex patterns like the [cake
+pattern](http://www.cakesolutions.net/teamblogs/2011/12/19/cake-pattern-in-depth),
+among other situations, are now efficiently supported.
+  
+In this example, we're going to try Zinc 1.0 to compile
+[ScalaTest](http://www.scalatest.org/). We're going to add a method to a widely
+referenced class: `AndHaveWord`, in file `Matcher.scala`.
+  
+[![scastie](/resources/img/blog/zinc-0.13-scalatest.gif)](/resources/img/blog/zinc-0.13-scalatest.gif)
+  
+With the previous incremental compiler, the compilation takes 21 seconds in an
+already hot compiler.
+  
+From a method addition, a pretty common operation in any codebase, Zinc 0.13.x
+adds all the source files that define a class that reference `AndHaveWord` and recompile them.
+So if we have several other class definitions those will also be recompiled.
+Then, Zinc will see the changes affected by those source files (transitively)
+and will not only recompile the users of `AndHaveWord`, but also any other
+source file that references to a symbol defined in the source file of the users.
+  
+The reason the above example takes 21 seconds to run is that Zinc analysis is
+done at the source level and not the class level.  This analysis, while correct,
+does not fit the common structure of Scala programs, where several class and
+objects can be defined in the same source file.
+  
+This, instead, is what happens when we make Zinc operate on the class-level:
+  
+[![scastie](/resources/img/blog/zinc-1.0-scalatest.gif)](/resources/img/blog/zinc-1.0-scalatest.gif)
+  
+In this scenario, class-based dependency does 7x better than the source-based.
+These speedups highly vary from codebase to codebase, where an array of several
+factors influence the result. In our experience, Zinc 1.0 behaves between 2x and
+22x better in many scenarios.
+  
+## Try it out!
+  
+Our commitment at the Scala Center is to make our work accessible to the
+Community so that the joy we have hacking on projects is also shared with the
+users.
+  
+Zinc 1.0 has historically been built upon the upcoming [sbt
+1.0](https://github.com/sbt/sbt), which makes it incompatible with the
+latest sbt version in production ([v0.13.13](https://github.com/sbt/sbt/tree/v0.13.13)).
+  
+In order to make it possible for any Scala developer to try the experimental
+Zinc 1.0 release, we have [created a sbt 0.13.13 plugin to compile with Zinc
+1.0](https://github.com/jvican/zinc/tree/sbt-plugin).
+  
+With this plugin, you can experience the improvements in your own projects by
+adding the following lines to your `project/plugins.sbt` file:
+  
+```
+resolvers += Resolver.bintrayRepo("scalacenter", "releases")
```

Comment:
Have you published the plugin? It seems empty on bintray https://bintray.com/scalacenter/releases/zinc